<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Wgrywanie pliku</string>
       </attribute>
       <widget class="QDateTimeEdit" name="dateTimeEdit">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>100</y>
          <width>194</width>
          <height>22</height>
         </rect>
        </property>
        <property name="displayFormat">
         <string>dd/MM/yyyy HH:mm:ss</string>
        </property>
       </widget>
       <widget class="QDateTimeEdit" name="dateTimeEdit_2">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>140</y>
          <width>194</width>
          <height>22</height>
         </rect>
        </property>
        <property name="displayFormat">
         <string>dd/MM/yyyy HH:mm:ss</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>100</y>
          <width>171</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Czas rozpoczęcia analizy</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>140</y>
          <width>171</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Czas zakończenia analizy</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>220</y>
          <width>221</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Średnia oddechy/min dla X</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>220</y>
          <width>211</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Średnia oddechy/min dla Y</string>
        </property>
       </widget>
       <widget class="QPushButton" name="startButton">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>340</y>
          <width>251</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Start analizy</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="textBrowser_Analysis">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>40</y>
          <width>291</width>
          <height>141</height>
         </rect>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_LB">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>191</width>
          <height>61</height>
         </rect>
        </property>
        <property name="title">
         <string>Rozpoczęcie działania aplikacji</string>
        </property>
        <widget class="QLabel" name="label_lbl">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>25</y>
           <width>151</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>220</y>
          <width>211</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Średnia oddechy/min dla Z</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>10</y>
          <width>211</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Informacja końcowa</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="textBrowser">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>250</y>
          <width>141</width>
          <height>51</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextBrowser" name="textBrowser_2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>250</y>
          <width>141</width>
          <height>51</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextBrowser" name="textBrowser_3">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>590</x>
          <y>250</y>
          <width>141</width>
          <height>51</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>841</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menupomoc">
    <property name="title">
     <string>Informacje</string>
    </property>
    <addaction name="actionInformacje_o_projekcie"/>
   </widget>
   <widget class="QMenu" name="menuPlik">
    <property name="title">
     <string>Plik</string>
    </property>
    <addaction name="actionOtw_rz_plik_2"/>
   </widget>
   <addaction name="menuPlik"/>
   <addaction name="menupomoc"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOtw_rz_plik">
   <property name="text">
    <string>Otwórz plik</string>
   </property>
  </action>
  <action name="actionOtw_rz_plik_2">
   <property name="text">
    <string>Otwórz plik</string>
   </property>
  </action>
  <action name="actionInformacje_o_projekcie">
   <property name="text">
    <string>Informacje o projekcie</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
